libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -897,6 +899,27 @@ class MapboxNavigation @VisibleForTesting internal constructor(
         routesPreviewController.changeRoutesPreviewPrimaryRoute(newPrimaryRoute)
     }
 
+    /**
+     * Convenience method that takes previewed routes and sets them to Navigator,
+     * clearing the previewed routes afterwards.
+     * Equivalent to:
+     * ```
+     * mapboxNavigation.setNavigationRoutes(mapboxNavigation.getRoutesPreview()!!.routesList)
+     * mapboxNavigation.setRoutesPreview(emptyList())
+     * ```
+     * @throws IllegalArgumentException when previewed routes are empty.
+     */
+    @Throws(IllegalArgumentException::class)
+    @ExperimentalPreviewMapboxNavigationAPI
+    fun moveRoutesFromPreviewToNavigator() {
```

Comment:
Not so sure about using `Navigator` here 🤔

Reply:
I can rename to `moveRoutesFromPreviewToActiveGuidance` or sth like that. @VysotskiVadim WDYT?

Reply:
if trip session isn't started, active guidance won't start.

Reply:
That's right but I agree that "Navigator" is an internal term. What about something with "actual"? Like `movePreviewedRoutesToActual`? Or do you have any other ideas?

Reply:
> Or do you have any other ideas?

`moveRoutesFromPreviewToNavigator` is my favourite option so far 🙂